openverse-api/catalog/api/utils/status_code_view.py

```diff
@@ -0,0 +1,27 @@
+from django.http import JsonResponse
+from django.utils.decorators import method_decorator
+from django.views.decorators.csrf import csrf_exempt
+from django.views.generic import View
+
+
+def get_status_code_view(data, status_code=200):
+    """
+    Get a class-based that returns the given data and status code on all HTTP
```

Comment:
```suggestion
    Get a class-based view that returns the given data and status code on all HTTP
```

Reply:
Fixed in 80fc446, couldn't accept suggestion because of 80 character limit.